fluid/neural_machine_translation/transformer/train.py

```diff
@@ -138,12 +144,14 @@ def test(exe):
         test_avg_costs = []
         for batch_id, data in enumerate(val_data()):
             if len(data) != TrainTaskConfig.batch_size:
```

Comment:
Please fix this

Reply:
Done. Refine the validation and use the global statistics.